### PR TITLE
Add gcp serviceaccount controller chart

### PR DESCRIPTION
**Pull request by rpahli**

### CODE REVIEW

File: charts/gcp-serviceaccount-controller/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: gcp-serviceaccount-controller
```
Comment (axdotl) [inline charts/gcp-serviceaccount-controller/Chart.yaml]: Thats not a meningful description.

File: charts/gcp-serviceaccount-controller/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+$ helm upgrade -i gcp-account-controller kiwigrid/gcp-serviceaccount-controller --namespace infra \
+    --set gcpCredentials="$(cat service-account.json | base64)"
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release-name` deployment:
+
+```console
+$ helm delete my-release-name
```
Comment (axdotl) [inline charts/gcp-serviceaccount-controller/README.md]: Add purge flag?

File: charts/gcp-serviceaccount-controller/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the Drupal chart and their default values.
```
Comment (axdotl) [inline charts/gcp-serviceaccount-controller/README.md]: Drupal?p

File: charts/gcp-serviceaccount-controller/templates/NOTES.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{{- if (eq "" .Values.gcpCredentials)}}
```
Comment (axdotl) [inline charts/gcp-serviceaccount-controller/templates/NOTES.txt]: A message to the user with a short howto would be great in any cases.

File: charts/gcp-serviceaccount-controller/templates/gcpserviceaccount.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
```
Comment (axdotl) [inline charts/gcp-serviceaccount-controller/templates/gcpserviceaccount.yaml]: Default Helm labels missing

File: charts/gcp-serviceaccount-controller/templates/rbac_role.yaml [hunk 6]
```diff
@@ -0,0 +1,67 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
```
Comment (axdotl) [inline charts/gcp-serviceaccount-controller/templates/rbac_role.yaml]: Default Helm labels missing

File: charts/gcp-serviceaccount-controller/templates/rbac_role_binding.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  creationTimestamp: null
+  name: manager-rolebinding
```
Comment (axdotl) [inline charts/gcp-serviceaccount-controller/templates/rbac_role_binding.yaml]: Default Helm labels missing

File: charts/gcp-serviceaccount-controller/templates/secret.yaml [hunk 8]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "gcp-serviceaccount-controller.fullname" . }}
```
Comment (axdotl) [inline charts/gcp-serviceaccount-controller/templates/secret.yaml]: Default Helm labels missing

File: charts/gcp-serviceaccount-controller/values.yaml [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+nameOverride: ""
+fullnameOverride: ""
+
+gcpCredentials: ""
+service:
+  type: ClusterIP
+  port: 80
+
+resources:
```
Comment (axdotl) [inline charts/gcp-serviceaccount-controller/values.yaml]: Should be disabled by default

File: charts/gcp-serviceaccount-controller/README.md (hunk 3 above)
Comment (monotek) [reply charts/gcp-serviceaccount-controller/README.md]: use "helm create"! ;-)

**Comment by monotek** [comment]
Please resolve merge conflicts...

File: charts/gcp-serviceaccount-controller/README.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+| Parameter                         | Description                             | Default                                                                                     |
+| --------------------------------- | --------------------------------------  | ---------------------------------------------------------                                   |
+| `image`                           | gcp service account controller image                          | `kiwigrid/gcp-serviceaccount-controller`                                                        |
+| `imageTag`                        | gcp service account controller image tag                      | `latest`                                                                                      |
```
Comment (monotek) [inline charts/gcp-serviceaccount-controller/README.md]: Don't use latest

File: charts/gcp-serviceaccount-controller/values.yaml [hunk 11]
```diff
@@ -0,0 +1,28 @@
+replicaCount: 1
+
+image:
+  repository: kiwigrid/gcp-serviceaccount-controller
+  tag: latest
```
Comment (monotek) [inline charts/gcp-serviceaccount-controller/values.yaml]: Don't use latest

**Pull request merged**